resolver-dns/src/main/java/io/netty/resolver/dns/DefaultDnsNameResolverBuilder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.resolver.dns;
+
+import io.netty.channel.EventLoop;
+
+/**
+ * Default {@link DefaultDnsNameResolverBuilder} that build a standard {@link DnsNameResolver}.
+ */
+public final class DefaultDnsNameResolverBuilder extends DnsNameResolverBuilder<DefaultDnsNameResolverBuilder> {
+
+    /**
+     * Creates a new builder.
+     *
+     * @param eventLoop the {@link EventLoop} the {@link EventLoop} which will perform the communication with the DNS
+     *                  servers.
+     */
+    public DefaultDnsNameResolverBuilder(EventLoop eventLoop) {
```

Comment:
I wonder if we want to make this `EventLoopGroup` as this way a user could pass either `EventLoopGroup` instance or `EventLoop` instance.